src/vma/dev/cq_mgr.h

```diff
@@ -111,11 +109,12 @@ class cq_mgr
 	friend class ring; // need to expose the m_n_global_sn only to ring 
 	friend class ring_simple; // need to expose the m_n_global_sn only to ring
 	friend class ring_bond; // need to expose the m_n_global_sn only to ring
+	friend class cq_mgr_mlx5;
```

Comment:
Without the friendship many functions and operands access restriction needed to be changed from private to protected.